web/src/features/menu/list/index.tsx

```diff
@@ -11,6 +11,8 @@ import React from 'react';
 
 export interface MenuItem {
   label: string;
+  progress?: number;
+  colorScheme?: string;
```

Comment:
The list menu is kinda built to be colourless, but since I'll be moving all the styling to scss eventually I wouldn't mind letting people pick the colour.

Though it should definitely have a default that matches the current colours of the menu.

Reply:
Added default of gray, matches the UI well. Forgot about having this default.

Regarding colors, I was thinking about adding a key for coloring each item individually, or just the progress bar and icon on their own, but didn't know if this would start bloating up the options.

Thoughts on color scheming system?

Reply:
> Thoughts on color scheming system?

I think `colorScheme` for progress bars is fine, *maybe* even `iconColor` but I wouldn't want people to be able to colour the whole buttons as I could see some horrible creations manifest out of that 😞

Reply:
True, I agree, could get a bit crazy.